src/pages/welcome.html

```diff
@@ -19,15 +19,17 @@
     <main class="section document">
       <div class="row">
         <div class="document-content">
-          <h1 class="document-headline ff-Met">Welcome to the Firefox Private Network Beta</h1>
-          <p class="lead">Thanks for participating in this limited-time, free beta of Firefox Private Network. You can use this lightweight browser extension to enhance your privacy and security on Firefox for desktop. Use it to:</p>
+          <h1 class="document-headline ff-Met">Welcome to the Firefox Private Network</h1>
+          <p class="lead">You can use this lightweight browser extension to enhance your privacy and security on Firefox for Desktop. Use it to:</p>
           <ol>
             <li>Increase your privacy on public Wi-Fi</li>
             <li>Mask your browsing history from your internet service provider</li>
             <li>Hide your location to reduce online tracking and data collection</li>
           </ol>
 
           <h2 class="ff-Met document-subhead">Your privacy</h2>
+          <p>Firefox is committed to your protecting your privacy. The browser-level protection, provided by Cloudflare, temporarily logs unidentified browsing history and deletes this data within 24 hours as a mechanism to detect and handle abuse on the network. Neither Firefox nor Cloudflare is able to associate usage with users as each party holds partial aspects of this data which are never joined.</p>
```

Comment:
First sentence has too many “your”s:
> to your protecting your privacy.

Reply:
I would like @betsymi to fix this double 'your's issue.

Reply:
Is this the last issue to resolve before we can merge?
If so, let’s just fix with “Firefox is committed to protecting your privacy.” (remove first “your”, since that’s the only way the sentence works). Betsy can open a new bug if she wants better wordsmithing.

Reply:
From @betsymi on Slack DM:

> Thanks for catching. It should say "Firefox is committed to protecting your privacy."

// @bakulf